encryption.md

```diff
@@ -58,8 +58,14 @@ TODO: Include language about safe nonce considerations.
 | ECDH-1PU+A256KW | P-521           | EC                 | ECDH-1PU key wrapping using key with NIST defined P-521 elliptic curve to create a 256 bits key as defined in [ecdh-1pu](https://tools.ietf.org/html/draft-madden-jose-ecdh-1pu-03#section-2.2)                                                         |
 | ECDH-1PU+A256KW | X25519          | OKP                | ECDH-1PU X25519 ([RFC7748 section 5](https://tools.ietf.org/html/rfc7748#section-5)) to create a 256 bits key as defined in [ecdh-1pu](https://tools.ietf.org/html/draft-madden-jose-ecdh-1pu-03#section-2.2)                                           |
 
-### Examples
+## Perfect Forward Secrecy
+
+Due to the triple Key Derivation algorithm used in ECDH-1PU and the ephemeral keys used per message in ECDH-ES, all messages sent via DIDComm have perfect forward secrecy without any additional security added by the transport layer. In ECDH-ES, PFS is obtained because the ephemeral key combined with the static recipient key produce a new derived symmetrical key for each message (which is then used to encrypt the content encryption key).
```

Comment:
This is only true for the sender. The recipient does not get PFS.

Reply:
This is called weak forward secrecy not perfect forward secrecy. We get perfect if both sides use ephemeral keys.

Reply:
Thanks for the feedback. I'll be sure to update to describe this better.